strapi/strapi
# chore(admin): convert Roles Settings

### What does it do?

* converts the roles section of settings to TS

### Why is it needed?

* TS conversion

### How to test it?

* Added some automated testing around the Permissions form
* Go to `Roles`
* create a new role
* save role
* Edit role
* save role
* check changes were persisted

### Related Issues

* resolves CONTENT-2086
